gradle.properties

```diff
@@ -39,10 +39,10 @@ nettyBoringSSLVersion=2.0.54.Final
 jacocoVersion=0.8.5
 protobufGradlePlugin=0.8.15
 protobufProtocVersion=3.21.7
-protobufUtilVersion=3.19.4
+protobufUtilVersion=3.24.3
```

Comment:
I see there is a PR on Pravega too for similar issue. Can we check the versions and try to keep them in sync just in order to avoid backward-forward incompatibility issue at a later stage?

Reply:
`protobufUtilVersion ` is not being used in pravega.

Reply:
```
protobufGradlePlugin=0.8.15
protobufProtocVersion=3.21.7
```

already syncing